install/kubernetes/cilium/values.yaml

```diff
@@ -1393,7 +1393,7 @@ clustermesh:
       pullPolicy: IfNotPresent
       # clustermesh-apiserver-digest
       digest: "sha256:e4a6d47d0d8e4ef52c44e99a906076e29e8513a35a972e60100df2168b381318"
-      useDigest: false
+      useDigest: true
```

Comment:
I think that we have discussed that this should be false by default for stable branches to avoid breaking any users deployment where people have their own images.